### ISSUE TITLE
"r.expand is not a function error" console error when adding a new row, within a Matrix

### ISSUE THREAD

**Issue by howells** [engram-design/SuperTable#372]
### Description

With 2.5.3 and 3.5.0-RC4, when I hit Add Row I see a console error, 

```
_input.js:284 Uncaught TypeError: r.expand is not a function
    at s.constructor.addRow (_input.js:284)
    at s.constructor.<anonymous> (garnish.js:893)
    at HTMLDivElement.o (jquery.js:10771)
    at HTMLDivElement.dispatch (jquery.js:5429)
    at HTMLDivElement.v.handle (jquery.js:5233)
    at Object.trigger (jquery.js:8715)
    at HTMLDivElement.<anonymous> (jquery.js:8793)
    at Function.each (jquery.js:381)
    at b.fn.init.each (jquery.js:203)
    at b.fn.init.trigger (jquery.js:8792)
```

However, the data does seem to save correctly.

### Steps to reproduce

1. Click Add Row on a super table field within a matrix

### Additional info

- Plugin version: 2.5.3
- Craft version: 3.5.0-RC4

**Comment by dennismeissner** [comment]
Same problem here. Actually this is happening in combination with neo in my case, which causes the neo spinner to never disappear again, which makes this issue critical for me.

**Comment by engram-design** [comment]
Fixed in [2.5.4](https://github.com/verbb/super-table/releases/tag/2.5.4)
